frontend/src/composables/use-analytics.ts

```diff
@@ -1,68 +1,7 @@
-import { computed, onMounted } from "vue"
 import { useContext } from "@nuxtjs/composition-api"
 
-import type { Events, EventName } from "~/types/analytics"
-import { useUiStore } from "~/stores/ui"
-import { useFeatureFlagStore } from "~/stores/feature-flag"
-
-import { log } from "~/utils/console"
-
-/**
- * The `ctx` parameter must be supplied if using this composable outside the
- * bounds of the composition API.
- */
 export const useAnalytics = () => {
```

Comment:
```suggestion
/**
 * This function is a thin wrapper around a plugin,
 * used to preserve tests and behavior from a prior
 * implementation. See `frontend/src/plugins/analytics.ts`
 * for the full analytics logic.
 */
export const useAnalytics = () => {
```

Nit: Some context might be useful here for future readers. Feel free to write your own copy, just a suggestion.

Reply:
Oh, yes definitely, thanks for the suggested docs.